packages/@aws-cdk/aws-codepipeline/README.md

```diff
@@ -1,29 +1,41 @@
-## AWS CodePipeline construct library
+## AWS CodePipeline Construct library
 
-Construct an empty pipeline:
+This library allows you to construct AWS CodePipelines.
+A Pipeline is modeled like other CDK Constructs:
+they form a tree, with Pipeline at the root.
+Stages then take a Pipeline as their parent,
+and finally Actions take a Stage as their parent.
+
+Example:
 
 ```ts
-const pipeline = new Pipeline(this, 'MyFirstPipeline');
+import codepipeline = require('@aws-cdk/aws-codepipeline');
+
+const pipeline = new codepipeline.Pipeline(this, 'MyFirstPipeline', {
+    pipelineName: 'MyFirstPipeline',
+});
+const stage = new codepipeline.Stage(pipeline, 'Source');
+new codepipeline.GithubSource(stage, 'GitHub Source', {
+    artifactName: 'SourceArtifact',
+    owner: 'awslabs',
+    repo: 'aws-cdk',
+    oauthToken: new SecretParameter(//...
+})
 ```
 
-All of the components of a pipeline are modeled as constructs.
+### Actions
 
-Append a stage to the pipeline:
+Actions are organized as follows.
 
-```ts
-const sourceStage = new Stage(pipeline, 'Source');
-```
+The most popular Actions for external integrations
```

Comment:
I wouldn't use the term "the most popular".
Also, the word "integration", which I know is used internally, is a bit confusing. I would just use something like "Pipeline actions for external services" and then, "Pipeline actions for AWS services"

Reply:
Will change.